src/oci.c

```diff
@@ -1800,6 +1673,11 @@ cc_oci_config_update (struct cc_oci_config *config,
 		state->mounts = NULL;
 	}
 
+	if(state->process) {
+		config->oci.process = *state->process;
```

Comment:
This looks like a memory leak as you are copying the dynamically-allocated `state->process` to `config`, but then just `NULL`-ing `state->process`.